source/vistas/core/bounds.py

```diff
@@ -21,7 +21,7 @@ def center(self) -> Vector3:
         ])
```

Comment:
So the `BoundingBox` class could be attributed to other kinds of bounding boxes. An `Extent` could produce a bounding box in 3D space (even though it doesn't now). It also made way for being able to add the `bounding_box.py` for implementing the `BoundingBoxHelper` class, which is a class for rendering the bounding boxes for any `Mesh`.

TL;DR - having both a `bounding_box.py` and a `bounds.py` in the `graphics` module didn't seem to fit very well, and since `BoundingBox` is more of a core concept anyway, not necessarily tied to graphics, I thought it fine to move up.

Reply:
Could you give me an example of using `BoundingBox` in a _non_-graphics context? Or are you planning to combine `Extent` and `BoundingBox` into a single class?

Reply:
I had not considered combining `Extent` and `BoundingBox`. They do enough different things that having them separate is good.

One thing I had thought of was having a `BoundingBox` for a selectable area. Here's the use case - for the Terrain & Color plugin, we could define bounding boxes for each of the features, and then allow the features to be highlighted when the user mouses over them. Then all the user has to do is click on that highlighted feature, and then they can view the zonal statistics for that region. This could also be used when defining the bounding box for a user-defined region to perform zonal statistics. All the while, the bounding box does not need to be rendered at all.

So, it's still a graphics-related context... perhaps there isn't a compelling reason to have it external to the graphics module.

Reply:
Yeah, it seems specific to graphics to me.